man/rnaseq_fdata.Rd

```diff
@@ -0,0 +1,19 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/data.R
+\name{rnaseq_fdata}
+\alias{rnaseq_fdata}
+\title{RNAseq Sample Metadata (f_data)}
+\format{
+A data.frame with 45 rows (samples) and 4 columns (sample identifier
+  and condition):
+\describe{
+  \item{SampleID}{Sample identifier (matches column headers in e_data)}
```

Comment:
rnaseq_fdata has "SampleName" as the column for me

Reply:
Fixed, thanks!